src/ng/filter/limitTo.js

```diff
@@ -9,9 +9,10 @@
  * Creates a new array or string containing only a specified number of elements. The elements
  * are taken from either the beginning or the end of the source array, string or number, as specified by
  * the value and sign (positive or negative) of `limit`. If a number is used as input, it is
- * converted to a string.
+ * converted to a string. Array-like values (e.g. NodeLists, jQuery objects, TypedArrays, Strings, etc)
+ * are also supported.
  *
- * @param {Array|string|number} input Source array, string or number to be limited.
+ * @param {Array} input Array or array-like to be limited.
```

Comment:
`orderBy` only accepts `{Array|ArrayLike}` (and calls `Array.prototype.map` on its input).